### ISSUE TITLE
Database Exception "Unknown column 'craft_navie_listitems.id' in 'field list'"

### ISSUE THREAD

**Issue by wfendler** [dutchheight/craft-navie#3]
Craft CMS 3.3.9 (headless mode)
Navite 1.1.3 (trial)

After installing I created a list and clicked the Navie CP link in the sidebar. I believe this takes you to the same place as "Manage items" takes you from `/admin/navie/settings/lists`. Neither of those links work.

Here's a copy of the stacktrace in case that's helpful:

```mysql
PDOException: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'craft_navie_listitems.id' in 'field list' in /var/www/html/vendor/yiisoft/yii2/db/Command.php:1290
Stack trace:
#0 /var/www/html/vendor/yiisoft/yii2/db/Command.php(1290): PDOStatement->execute()
#1 /var/www/html/vendor/yiisoft/yii2/db/Command.php(1158): yii\db\Command->internalExecute('SELECT EXISTS(S...')
#2 /var/www/html/vendor/yiisoft/yii2/db/Command.php(425): yii\db\Command->queryInternal('fetchColumn', 0)
#3 /var/www/html/vendor/yiisoft/yii2/db/Query.php(425): yii\db\Command->queryScalar()
#4 /var/www/html/vendor/craftcms/cms/src/db/Query.php(223): yii\db\Query->exists(NULL)
#5 /var/www/html/vendor/craftcms/cms/src/elements/db/ElementQuery.php(1515): craft\db\Query->exists(NULL)
#6 /var/www/html/vendor/twig/twig/src/Extension/CoreExtension.php(1495): craft\elements\db\ElementQuery->exists()
#7 /var/www/html/vendor/craftcms/cms/src/helpers/Template.php(105): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(dutchheight\navie\elements\db\ListItemQuery), 'exists', Array, 'method', false, false)
#8 /var/www/html/storage/runtime/compiled_templates/e7/e796369c8cdf1144bfdb05b673bc01a00135549ee325b7787ab061ae629580f1.php(131): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(dutchheight\navie\elements\db\ListItemQuery), 'exists', Array, 'method')
#9 /var/www/html/vendor/twig/twig/src/Template.php(407): __TwigTemplate_69a8c93587d28d813c074296705ce226b8029b95cbee64c972384fadff8c5702->doDisplay(Array, Array)
#10 /var/www/html/vendor/twig/twig/src/Template.php(380): Twig\Template->displayWithErrorHandling(Array, Array)
#11 /var/www/html/storage/runtime/compiled_templates/41/41204bdf847a9e5736c0fcd31ed179ee3fe82bc9627462130c5a6fe8b4bb69ec.php(140): Twig\Template->display(Array)
#12 /var/www/html/vendor/twig/twig/src/Template.php(184): __TwigTemplate_1fc5c49add9d3efe46fdbbf36ca00efa963322141af3d0b4d6bb8ae39b74916d->block_content(Array, Array)
#13 /var/www/html/storage/runtime/compiled_templates/5a/5ad5d889cb3dc2b0ca35f21d1ce8b375a83ff3adf985e399069496a5670affdb.php(727): Twig\Template->displayBlock('content', Array, Array)
#14 /var/www/html/vendor/twig/twig/src/Template.php(184): __TwigTemplate_7db007bf09ebc06344d30dbebcbb9f77b03a95ff79ebb515bba74d83f904e33d->block_main(Array, Array)
#15 /var/www/html/storage/runtime/compiled_templates/5a/5ad5d889cb3dc2b0ca35f21d1ce8b375a83ff3adf985e399069496a5670affdb.php(576): Twig\Template->displayBlock('main', Array, Array)
#16 /var/www/html/vendor/twig/twig/src/Template.php(184): __TwigTemplate_7db007bf09ebc06344d30dbebcbb9f77b03a95ff79ebb515bba74d83f904e33d->block_body(Array, Array)
#17 /var/www/html/storage/runtime/compiled_templates/c0/c0466ed6dbd0819cef51a8265c2aad768d1c6584e4ab7efe88031ba61d59b469.php(75): Twig\Template->displayBlock('body', Array, Array)
#18 /var/www/html/vendor/twig/twig/src/Template.php(407): __TwigTemplate_bd9738498657cb4b4cd092130af69463215f88d4ca49f6e779aeb5efe1069924->doDisplay(Array, Array)
#19 /var/www/html/vendor/twig/twig/src/Template.php(380): Twig\Template->displayWithErrorHandling(Array, Array)
#20 /var/www/html/storage/runtime/compiled_templates/22/22a5c4fc766c6c06d859f8e6b8aa0cb65802ea714530ea0269c2fe550fd7abf3.php(56): Twig\Template->display(Array, Array)
#21 /var/www/html/vendor/twig/twig/src/Template.php(407): __TwigTemplate_a89b1560e45c772d9b642c3735c1556fdaeecef3930c8f578a296595f7ebff9a->doDisplay(Array, Array)
#22 /var/www/html/vendor/twig/twig/src/Template.php(380): Twig\Template->displayWithErrorHandling(Array, Array)
#23 /var/www/html/storage/runtime/compiled_templates/5a/5ad5d889cb3dc2b0ca35f21d1ce8b375a83ff3adf985e399069496a5670affdb.php(118): Twig\Template->display(Array, Array)
#24 /var/www/html/vendor/twig/twig/src/Template.php(407): __TwigTemplate_7db007bf09ebc06344d30dbebcbb9f77b03a95ff79ebb515bba74d83f904e33d->doDisplay(Array, Array)
#25 /var/www/html/vendor/twig/twig/src/Template.php(380): Twig\Template->displayWithErrorHandling(Array, Array)
#26 /var/www/html/storage/runtime/compiled_templates/41/41204bdf847a9e5736c0fcd31ed179ee3fe82bc9627462130c5a6fe8b4bb69ec.php(71): Twig\Template->display(Array, Array)
#27 /var/www/html/vendor/twig/twig/src/Template.php(407): __TwigTemplate_1fc5c49add9d3efe46fdbbf36ca00efa963322141af3d0b4d6bb8ae39b74916d->doDisplay(Array, Array)
#28 /var/www/html/vendor/twig/twig/src/Template.php(380): Twig\Template->displayWithErrorHandling(Array, Array)
#29 /var/www/html/storage/runtime/compiled_templates/f4/f4fbe6d6d8b9f11f151419c977093ad11ae2c82669222ed9b50bf1bea5b6ca21.php(55): Twig\Template->display(Array, Array)
#30 /var/www/html/vendor/twig/twig/src/Template.php(407): __TwigTemplate_81eff19daa15eb4cec6274142a4adac3afdab014ccf6f12e2fb4f2609f5d97bb->doDisplay(Array, Array)
#31 /var/www/html/vendor/twig/twig/src/Template.php(380): Twig\Template->displayWithErrorHandling(Array, Array)
#32 /var/www/html/vendor/twig/twig/src/Template.php(392): Twig\Template->display(Array)
#33 /var/www/html/vendor/twig/twig/src/TemplateWrapper.php(45): Twig\Template->render(Array, Array)
#34 /var/www/html/vendor/twig/twig/src/Environment.php(318): Twig\TemplateWrapper->render(Array)
#35 /var/www/html/vendor/craftcms/cms/src/web/View.php(344): Twig\Environment->render('navie/listitems...', Array)
#36 /var/www/html/vendor/craftcms/cms/src/web/View.php(393): craft\web\View->renderTemplate('navie/listitems...', Array)
#37 /var/www/html/vendor/craftcms/cms/src/web/Controller.php(243): craft\web\View->renderPageTemplate('navie/listitems...', Array)
#38 /var/www/html/vendor/dutchheight/craft-navie/src/controllers/ListsController.php(198): craft\web\Controller->renderTemplate('navie/listitems...', Array)
#39 [internal function]: dutchheight\navie\controllers\ListsController->actionListItemIndex(NULL)
#40 /var/www/html/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#41 /var/www/html/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#42 /var/www/html/vendor/craftcms/cms/src/web/Controller.php(187): yii\base\Controller->runAction('list-item-index', Array)
#43 /var/www/html/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('list-item-index', Array)
#44 /var/www/html/vendor/craftcms/cms/src/web/Application.php(299): yii\base\Module->runAction('navie/lists/lis...', Array)
#45 /var/www/html/vendor/yiisoft/yii2/web/Application.php(103): craft\web\Application->runAction('navie/lists/lis...', Array)
#46 /var/www/html/vendor/craftcms/cms/src/web/Application.php(284): yii\web\Application->handleRequest(Object(craft\web\Request))
#47 /var/www/html/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#48 /var/www/html/web/index.php(21): yii\base\Application->run()
#49 {main}

Next yii\db\Exception: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'craft_navie_listitems.id' in 'field list'
The SQL being executed was: SELECT EXISTS(SELECT `elements`.`id`, `elements`.`fieldLayoutId`, `elements`.`uid`, `elements`.`enabled`, `elements`.`archived`, `elements`.`dateCreated`, `elements`.`dateUpdated`, `elements_sites`.`slug`, `elements_sites`.`siteId`, `elements_sites`.`enabled` AS `enabledForSite`, `craft_navie_listitems`.`id`, `craft_navie_listitems`.`listId`, `craft_navie_listitems`.`elementId`, `craft_navie_listitems`.`type`, `craft_navie_listitems`.`url`, `craft_navie_listitems`.`target`, `linked_elements`.`uri` AS `linkedElementUrl`, `content`.`id` AS `contentId`, `content`.`title`, `structureelements`.`root`, `structureelements`.`lft`, `structureelements`.`rgt`, `structureelements`.`level`, `structureelements`.`structureId`, `elements`.`draftId`, `drafts`.`sourceId` AS `draftSourceId`, `drafts`.`creatorId` AS `draftCreatorId`, `drafts`.`name` AS `draftName`, `drafts`.`notes` AS `draftNotes`
FROM (SELECT `elements`.`id` AS `elementsId`, `elements_sites`.`id` AS `elementsSitesId`, `content`.`id` AS `contentId`, `structureelements`.`structureId`
FROM `craft_elements` `elements`
INNER JOIN `craft_navie_listitems` `navie_listitems` ON `navie_listitems`.`id` = `elements`.`id`
INNER JOIN `craft_elements_sites` `elements_sites` ON `elements_sites`.`elementId` = `elements`.`id`
INNER JOIN `craft_content` `content` ON (`content`.`elementId` = `elements`.`id`) AND (`content`.`siteId` = `elements_sites`.`siteId`)
LEFT JOIN `craft_structureelements` `structureelements` ON (`structureelements`.`elementId` = `elements`.`id`) AND (EXISTS (SELECT *
FROM `craft_structures`
WHERE (`id` = `structureelements`.`structureId`) AND (`dateDeleted` IS NULL)))
INNER JOIN `craft_drafts` `drafts` ON `drafts`.`id` = `elements`.`draftId`
WHERE (`elements_sites`.`siteId`='2') AND (`elements`.`archived`=FALSE) AND (`elements`.`dateDeleted` IS NULL) AND (`drafts`.`sourceId` IS NULL) AND (`elements`.`revisionId` IS NULL)
ORDER BY `structureelements`.`lft`, `elements`.`dateCreated` DESC) `subquery`
INNER JOIN `craft_navie_listitems` `navie_listitems` ON `navie_listitems`.`id` = `subquery`.`elementsId`
LEFT JOIN `craft_elements_sites` `linked_elements` ON `navie_listitems`.`elementId` = `linked_elements`.`elementId` AND linked_elements.siteId = 2
INNER JOIN `craft_elements` `elements` ON `elements`.`id` = `subquery`.`elementsId`
INNER JOIN `craft_elements_sites` `elements_sites` ON `elements_sites`.`id` = `subquery`.`elementsSitesId`
INNER JOIN `craft_content` `content` ON `content`.`id` = `subquery`.`contentId`
LEFT JOIN `craft_structureelements` `structureelements` ON (`structureelements`.`elementId` = `subquery`.`elementsId`) AND (`structureelements`.`structureId` = `subquery`.`structureId`)
INNER JOIN `craft_drafts` `drafts` ON `drafts`.`id` = `elements`.`draftId`
ORDER BY `structureelements`.`lft`, `elements`.`dateCreated` DESC) in /var/www/html/vendor/yiisoft/yii2/db/Schema.php:664
Stack trace:
#0 /var/www/html/vendor/yiisoft/yii2/db/Command.php(1295): yii\db\Schema->convertException(Object(PDOException), 'SELECT EXISTS(S...')
#1 /var/www/html/vendor/yiisoft/yii2/db/Command.php(1158): yii\db\Command->internalExecute('SELECT EXISTS(S...')
#2 /var/www/html/vendor/yiisoft/yii2/db/Command.php(425): yii\db\Command->queryInternal('fetchColumn', 0)
#3 /var/www/html/vendor/yiisoft/yii2/db/Query.php(425): yii\db\Command->queryScalar()
#4 /var/www/html/vendor/craftcms/cms/src/db/Query.php(223): yii\db\Query->exists(NULL)
#5 /var/www/html/vendor/craftcms/cms/src/elements/db/ElementQuery.php(1515): craft\db\Query->exists(NULL)
#6 /var/www/html/vendor/twig/twig/src/Extension/CoreExtension.php(1495): craft\elements\db\ElementQuery->exists()
#7 /var/www/html/vendor/craftcms/cms/src/helpers/Template.php(105): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(dutchheight\navie\elements\db\ListItemQuery), 'exists', Array, 'method', false, false)
#8 /var/www/html/storage/runtime/compiled_templates/e7/e796369c8cdf1144bfdb05b673bc01a00135549ee325b7787ab061ae629580f1.php(131): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(dutchheight\navie\elements\db\ListItemQuery), 'exists', Array, 'method')
#9 /var/www/html/vendor/twig/twig/src/Template.php(407): __TwigTemplate_69a8c93587d28d813c074296705ce226b8029b95cbee64c972384fadff8c5702->doDisplay(Array, Array)
#10 /var/www/html/vendor/twig/twig/src/Template.php(380): Twig\Template->displayWithErrorHandling(Array, Array)
#11 /var/www/html/storage/runtime/compiled_templates/41/41204bdf847a9e5736c0fcd31ed179ee3fe82bc9627462130c5a6fe8b4bb69ec.php(140): Twig\Template->display(Array)
#12 /var/www/html/vendor/twig/twig/src/Template.php(184): __TwigTemplate_1fc5c49add9d3efe46fdbbf36ca00efa963322141af3d0b4d6bb8ae39b74916d->block_content(Array, Array)
#13 /var/www/html/storage/runtime/compiled_templates/5a/5ad5d889cb3dc2b0ca35f21d1ce8b375a83ff3adf985e399069496a5670affdb.php(727): Twig\Template->displayBlock('content', Array, Array)
#14 /var/www/html/vendor/twig/twig/src/Template.php(184): __TwigTemplate_7db007bf09ebc06344d30dbebcbb9f77b03a95ff79ebb515bba74d83f904e33d->block_main(Array, Array)
#15 /var/www/html/storage/runtime/compiled_templates/5a/5ad5d889cb3dc2b0ca35f21d1ce8b375a83ff3adf985e399069496a5670affdb.php(576): Twig\Template->displayBlock('main', Array, Array)
#16 /var/www/html/vendor/twig/twig/src/Template.php(184): __TwigTemplate_7db007bf09ebc06344d30dbebcbb9f77b03a95ff79ebb515bba74d83f904e33d->block_body(Array, Array)
#17 /var/www/html/storage/runtime/compiled_templates/c0/c0466ed6dbd0819cef51a8265c2aad768d1c6584e4ab7efe88031ba61d59b469.php(75): Twig\Template->displayBlock('body', Array, Array)
#18 /var/www/html/vendor/twig/twig/src/Template.php(407): __TwigTemplate_bd9738498657cb4b4cd092130af69463215f88d4ca49f6e779aeb5efe1069924->doDisplay(Array, Array)
#19 /var/www/html/vendor/twig/twig/src/Template.php(380): Twig\Template->displayWithErrorHandling(Array, Array)
#20 /var/www/html/storage/runtime/compiled_templates/22/22a5c4fc766c6c06d859f8e6b8aa0cb65802ea714530ea0269c2fe550fd7abf3.php(56): Twig\Template->display(Array, Array)
#21 /var/www/html/vendor/twig/twig/src/Template.php(407): __TwigTemplate_a89b1560e45c772d9b642c3735c1556fdaeecef3930c8f578a296595f7ebff9a->doDisplay(Array, Array)
#22 /var/www/html/vendor/twig/twig/src/Template.php(380): Twig\Template->displayWithErrorHandling(Array, Array)
#23 /var/www/html/storage/runtime/compiled_templates/5a/5ad5d889cb3dc2b0ca35f21d1ce8b375a83ff3adf985e399069496a5670affdb.php(118): Twig\Template->display(Array, Array)
#24 /var/www/html/vendor/twig/twig/src/Template.php(407): __TwigTemplate_7db007bf09ebc06344d30dbebcbb9f77b03a95ff79ebb515bba74d83f904e33d->doDisplay(Array, Array)
#25 /var/www/html/vendor/twig/twig/src/Template.php(380): Twig\Template->displayWithErrorHandling(Array, Array)
#26 /var/www/html/storage/runtime/compiled_templates/41/41204bdf847a9e5736c0fcd31ed179ee3fe82bc9627462130c5a6fe8b4bb69ec.php(71): Twig\Template->display(Array, Array)
#27 /var/www/html/vendor/twig/twig/src/Template.php(407): __TwigTemplate_1fc5c49add9d3efe46fdbbf36ca00efa963322141af3d0b4d6bb8ae39b74916d->doDisplay(Array, Array)
#28 /var/www/html/vendor/twig/twig/src/Template.php(380): Twig\Template->displayWithErrorHandling(Array, Array)
#29 /var/www/html/storage/runtime/compiled_templates/f4/f4fbe6d6d8b9f11f151419c977093ad11ae2c82669222ed9b50bf1bea5b6ca21.php(55): Twig\Template->display(Array, Array)
#30 /var/www/html/vendor/twig/twig/src/Template.php(407): __TwigTemplate_81eff19daa15eb4cec6274142a4adac3afdab014ccf6f12e2fb4f2609f5d97bb->doDisplay(Array, Array)
#31 /var/www/html/vendor/twig/twig/src/Template.php(380): Twig\Template->displayWithErrorHandling(Array, Array)
#32 /var/www/html/vendor/twig/twig/src/Template.php(392): Twig\Template->display(Array)
#33 /var/www/html/vendor/twig/twig/src/TemplateWrapper.php(45): Twig\Template->render(Array, Array)
#34 /var/www/html/vendor/twig/twig/src/Environment.php(318): Twig\TemplateWrapper->render(Array)
#35 /var/www/html/vendor/craftcms/cms/src/web/View.php(344): Twig\Environment->render('navie/listitems...', Array)
#36 /var/www/html/vendor/craftcms/cms/src/web/View.php(393): craft\web\View->renderTemplate('navie/listitems...', Array)
#37 /var/www/html/vendor/craftcms/cms/src/web/Controller.php(243): craft\web\View->renderPageTemplate('navie/listitems...', Array)
#38 /var/www/html/vendor/dutchheight/craft-navie/src/controllers/ListsController.php(198): craft\web\Controller->renderTemplate('navie/listitems...', Array)
#39 [internal function]: dutchheight\navie\controllers\ListsController->actionListItemIndex(NULL)
#40 /var/www/html/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#41 /var/www/html/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#42 /var/www/html/vendor/craftcms/cms/src/web/Controller.php(187): yii\base\Controller->runAction('list-item-index', Array)
#43 /var/www/html/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('list-item-index', Array)
#44 /var/www/html/vendor/craftcms/cms/src/web/Application.php(299): yii\base\Module->runAction('navie/lists/lis...', Array)
#45 /var/www/html/vendor/yiisoft/yii2/web/Application.php(103): craft\web\Application->runAction('navie/lists/lis...', Array)
#46 /var/www/html/vendor/craftcms/cms/src/web/Application.php(284): yii\web\Application->handleRequest(Object(craft\web\Request))
#47 /var/www/html/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#48 /var/www/html/web/index.php(21): yii\base\Application->run()
#49 {main}
Additional Information:
Array
(
    [0] => 42S22
    [1] => 1054
    [2] => Unknown column 'craft_navie_listitems.id' in 'field list'
)
```

**Comment by freimutt** [comment]
Fixed in f8f1004
Could you try to update to the latest version.

**Comment by wfendler** [comment]
Just updated. Looks like it works now, thanks!
